index.js

```diff
@@ -79,7 +98,14 @@ document.getElementById('start_button').addEventListener('click', (e) => {
 document.getElementById('reveal_button').addEventListener('click', (e) => {
     document.getElementById('answer').style.display ='block'
     document.getElementById('next_question_button').disabled = false
+    document.getElementById('hint_button').disabled = true
+    document.getElementById('reveal_button').disabled = true
     let submit_button = document.getElementById('submit_button')
-    submit_button.disabled = true;
-    submit_button.style.cursor = 'default'
+    submit_button.disabled = true
+})
+
+document.getElementById('hint_button').addEventListener('click', (e) => {
+    document.getElementById('hint_button').disabled = true
+    document.getElementById('hint').style.display = 'block'
```

Comment:
This is being repeated now, once in the event listener and then again in get_hint(). Suggest removing it from here and keeping the one in the get_hint() function